test/test_parser.rb

```diff
@@ -10241,6 +10241,30 @@ def test_hash_pair_value_omission
         | ~~~~~ expression (pair)},
       SINCE_3_1)
 
+    assert_parses(
+      s(:begin,
+        s(:lvasgn, :foo,
+          s(:int, 1)),
+        s(:hash,
+          s(:pair,
+            s(:sym, :foo),
+            s(:lvar, :foo)))),
+      %q{foo = 1; {foo:}},
+      %q{},
+      SINCE_3_1)
+
+      assert_parses(
+        s(:begin,
+          s(:lvasgn, :_foo,
+            s(:int, 1)),
+          s(:hash,
+            s(:pair,
+              s(:sym, :_foo),
+              s(:lvar, :_foo)))),
+        %q{_foo = 1; {_foo:}},
+        %q{},
+        SINCE_3_1)
```

Comment:
That's just my two cents. Everything else looks good.

```suggestion
    assert_parses(
      s(:begin,
        s(:lvasgn, :_foo,
          s(:int, 1)),
        s(:hash,
          s(:pair,
            s(:sym, :_foo),
            s(:lvar, :_foo)))),
      %q{_foo = 1; {_foo:}},
      %q{},
      SINCE_3_1)
```

Reply:
Thanks, done